internal/accountstable/accountstable.go

```diff
@@ -112,7 +108,8 @@ func MintAurumUpdateAccountBalanceTable(dbConnection *sql.DB, pkhash []byte, val
 
 func GetBalance(dbConnection *sql.DB, pkhash []byte) (uint64, error) {
 	// search for pkhash's balance
-	row, err := dbConnection.Query(sqlstatements.GET_BALANCE_FROM_ACCOUNT_BALANCES_BY_PUB_KEY_HASH + hex.EncodeToString(pkhash) + "\"")
+	// row, err := dbConnection.Query(sqlstatements.GET_BALANCE_FROM_ACCOUNT_BALANCES_BY_PUB_KEY_HASH + hex.EncodeToString(pkhash) + "\"")
```

Comment:
Purpose of this comment?

Reply:
Shoot! I always forget the small commented out code that I use as a reference. I will make sure to look over that more often

Reply:
I'll go back to how it was before refactoring my bad, didn't notice it as an issue